napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -1243,13 +1243,20 @@ def get_mac_address_table(self):
         * 16       0050.56bb.0164    dynamic      -       F    F    po2
         * 13       90e2.ba5a.9f30    dynamic      -       F    F    eth1/2
         * 13       90e2.ba4b.fc78    dynamic      -       F    F    eth1/1
+          39       0100.5e00.4b4b    igmp         0       F    F    Po1 Po2 Po22
+          110      0100.5e00.0118    igmp         0       F    F    Po1 Po2
+                                                                    Eth142/1/3 Eth112/1/5
+                                                                    Eth112/1/6 Eth122/1/5
+
         """
 
         #  The '*' is stripped out later
         RE_MACTABLE_FORMAT1 = r"^\s+{}\s+{}\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+".format(VLAN_REGEX,
                                                                                   MAC_REGEX)
         RE_MACTABLE_FORMAT2 = r"^\s+{}\s+{}\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+".format('-',
                                                                                   MAC_REGEX)
+        # REGEX dedicated for lines with only interfaces (suite of the previous MAC address)
+        RE_MACTABLE_FORMAT3 = r"^\s+"
```

Comment:
I think you would need to expand this to at least:

RE_MACTABLE_FORMAT3 = r"\s+\S+"

Reply:
Yes you're right, I'll change it